packages/node/src/indexer/indexer.manager.ts

```diff
@@ -345,13 +350,15 @@ export class IndexerManager {
     for (const handler of ds.mapping.handlers) {
       const processor = plugin.handlerProcessors[handler.kind];
       if (isBlockHandlerProcessor(processor)) {
-        const transformedOutput = processor.transformer(block, ds);
+        const transformedOutput = await processor.transformer(
+          block,
+          ds,
+          this.api,
```

Comment:
This exposes the whole api to the ds-sandbox which is currently needed to get the eth block hash. A user could easily modify the ds-processor and effectively have full use of the api in here then pass what they need to their project handlers.

 Not sure how to address. We could just not provide the ethereum block hash until we offer an ethereum wrapper around the api